src/navigation/Index.tsx

```diff
@@ -21,7 +21,7 @@ export default class Navigation extends Component {
   render() {
     return (
       <NavigationContainer theme={NavigationContainerTheme}>
-        <Stack.Navigator initialRouteName='SignUp'>
+        <Stack.Navigator initialRouteName='MyFlights'>
```

Comment:
The `<Stack.Navigator initialRouteName='SignUp'>` component should be removed since it doesn't add any style